### DISCUSSION TITLE
output clipped, log limit 1MiB reached error

### DISCUSSION THREAD

**Discussion by dongshanlang** [apache/superset#14280] [General]
when I run :
`docker build  --target superset-node -t supserset-node:1.0 .`
the result would be:
[webpack.Progress] 69% building 3513/3547 modules 34 active /app/superset-frontend/node_modules/rc-picker/es/hooks/useCellClassName.js
[webpack.Progress] 69% building 3514/3547 modules 33 active /app/superset-frontend/node_modules/rc-picker/es/hooks/useCellClassName.js
[webpack.Progress] 69% bu
 [output clipped, log limit 1MiB reached]
------------
What's the problem?

**Comment by BigRantLing** [comment]
I have the same issue. Did you fix it ?
